### ISSUE TITLE
Multiple position entries - same date / timestamp

### ISSUE THREAD

**Issue by m-menon** [tananaev/traccar#1807]
Any idea what could be causing these? There are multiple entries. Btw a related question - is it possible to set the server config in such a way that if the trackers send position data more than once in 30 seconds, then ignore it? I mean I would not want a tracker to send data like 10 times in a minute, that would be overloading the server and unnecessary. Any suggestion

Thank you so much,
M.M

**Comment by tananaev** [comment]
Usually this happens because device doesn't have a GPS fix and it reports old location.

There is no way to limit number of positions per period of time, but it's possible to filter positions with same time. Use `filter.distance` config parameter for it:

https://www.traccar.org/configuration-file/

**Comment by m-menon** [comment]
hi,

When you say filter positions is it for display only? Or does this also affect what goes into the table?

Thanks,
M.M

**Comment by tananaev** [comment]
It does affect what goes into the table. Filtered positions won't be stored in the database.

**Comment by m-menon** [comment]
in that case would `filter.duplicate` be better cos it seems to filter duplicate based on timestamp?

**Comment by 4nonym0us** [comment]
Additionally, I'd like to ask why don't traccar uses **ServerTime** instead of **FixTime**?
ServerTime can stored in ISO format (including timezone) so we can use this time everywhere in app?

**Comment by m-menon** [comment]
There should be good reasons for that I guess. Some trackers actually store the position information when network is not available and post them later once network is available. In that case the server will use ServerTime which is totally wrong - the positions were recorded at a different time

**Comment by 4nonym0us** [comment]
That's a reasonable argument. I agree that using FixTime is fine.

**Comment by tananaev** [comment]
I actually meant `filter.duplicate` parameter.

**Comment by m-menon** [comment]
Ok. Can I set this key filter.limit to say 28 seconds, then those position updates less than 28 should get filtered out I guess? Thanks

**Comment by tananaev** [comment]
Yes, correct.

**Comment by m-menon** [comment]
okay thanks :+1:

**Comment by m-menon** [comment]
Btw just wanted to confirm 
filter.invalid	Filter invalid (valid field is set to false) positions.

is that a boolean value? So it ignores those gps sentences that have the valid flag set to false?

Thanks

**Comment by tananaev** [comment]
Yes, correct.

**Comment by 4nonym0us** [comment]
Should we enable the **filter.enable** (from docs: Boolean flag to enable or disable position filtering.) in order to make all of the **filter._** entries work?

**Comment by tananaev** [comment]
Yes, you need to set `filter.enable` to `true` for any filters to work.

**Comment by 4nonym0us** [comment]
Thanks a lot for the answers, @tananaev!
